RHIDP-2024: Configuring PostgreSQL databases in RHDH (#262)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/assembly-configuring-external-postgresql-databases.adoc

[id='assembly-configuring-external-postgresql-databases_{context}']
= Configuring external PostgreSQL databases

> As an administrator, you can configure and use external PostgreSQL databases in {product}. Based on your needs, you can use a PostgreSQL certificate file to configure an external PostgreSQL instance using the Operator or Helm Chart.

Reviewer: ```suggestion
As an administrator, you can configure and use external PostgreSQL databases in {product}. You can use a PostgreSQL certificate file to configure an external PostgreSQL instance using the Operator or Helm Chart.
```

[NOTE]
====

> {product-very-short} supports only configuring external PostgreSQL databases. You are responsible for maintenance activities, such as backing up your data or configuring high availability (HA) for the external PostgreSQL databases.

Reviewer: ```suggestion
{product-short} supports only configuring external PostgreSQL databases. You can perform maintenance activities, such as backing up your data or configuring high availability (HA) for the external PostgreSQL databases.
```
